recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "1.2.1" %}
+{% set version = "2.0.1" %}
 
 package:
   name: altair
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/a/altair/altair-{{ version }}.tar.gz
-  sha256: c1303f77f1ba4d632f2958c83c0f457b2b969860b1ac9adfb872aefa1780baa7
```

Comment:
@jakevdp I switched back to PyPI b/c the checksum is stable, GitHub changes it if there is an change in the version tag.